xyz-hub-service/src/main/java/com/here/xyz/hub/rest/TagApi.java

```diff
@@ -127,6 +133,10 @@ public static Future<Tag> updateTag(Marker marker, String spaceId, String tagId,
       return Future.failedFuture(new ValidationException("Invalid version parameter"));
     }
 
+    if (!Tag.isDescriptionValid(description)) {
+      return Future.failedFuture(new ValidationException("Invalid description parameter"));
```

Comment:
ValidationExceptions are user-facing. It would be good to explain what exactly is the issue. E.g. explain the rules (max length, ...)